front/views/settings/NotificationView.tsx

```diff
@@ -1,36 +1,107 @@
 import React from "react";
-import { View } from 'react-native';
+import { View } from "react-native";
 import { Center, Button, Text, Switch, Heading } from "native-base";
-import { translate } from "../../i18n/i18n";
+import { translate, Translate } from "../../i18n/i18n";
+import { useDispatch } from "react-redux";
+import { RootState, useSelector } from "../../state/Store";
+import { useLanguage } from "../../state/LanguageSlice";
+import { SettingsState, updateSettings } from "../../state/SettingsSlice";
+import ElementList from "../../components/GtkUI/ElementList";
 
-const NotificationsView = ({navigation}) => {
-    return (
-        <Center style={{ flex: 1, justifyContent: 'center' }}>
+const NotificationsView = ({ navigation }) => {
+	const dispatch = useDispatch();
+	let settings: SettingsState = useSelector(
+		(state: RootState) => state.settings
+	);
+	const [pushNotifications, setPushNotifications] = React.useState(
```

Comment:
OK i used it because the state wasn't updates but it turns out it was because the useSelector was wrong (state.settings instead of state.settings.settings)